anvio/synteny.py

```diff
@@ -307,7 +307,8 @@ def annotate_window(self, window):
         """This method annotates a gene-callers-id window
 
         This method will annotate a gene-callers-id window based using annotation sources provided
-        by the user (e.g. COGs, pan_db)
+        by the user (e.g. COGs, pan_db). If the user provided the `take_first_cog` flag, the COG
```

Comment:
I think the docstring here needs an update since that flag does not exist anymore ;)

other than that, I think this is good to go. please merge it whenever you like, @mschecht. thank you very much!

Reply:
Thanks!